openjdk/playlist.xml

```diff
@@ -2724,6 +2724,11 @@
 	</test>
 	<test>
 		<testCaseName>jdk_vector_float128_j9</testCaseName>
+		<disables>
+			<disable>
+				<comment>https://github.com/eclipse-openj9/openj9/issues/21945</comment>
```

Comment:
Should we only exclude vector tests for JDK25? `<version>25</version>`

Reply:
Yes! Thank you. Updated it.